queries/tags.scm

```diff
@@ -1,46 +1,23 @@
-(class_declaration
- name: (identifier) @name
- ) @definition.class
-
-(class_declaration
-   bases: (base_list (_) @name)
- ) @reference.class
-
-(interface_declaration
- name: (identifier) @name
- ) @definition.interface
-
-(interface_declaration
- bases: (base_list (_) @name)
- ) @reference.interface
-
-(method_declaration
- name: (identifier) @name
- ) @definition.method
-
-(object_creation_expression
- type: (identifier) @name
- ) @reference.class
-
-(type_parameter_constraints_clause
- target: (identifier) @name
- ) @reference.class
-
-(type_constraint
- type: (identifier) @name
- ) @reference.class
-
-(variable_declaration
- type: (identifier) @name
- ) @reference.class
-
-(invocation_expression
- function:
-  (member_access_expression
-    name: (identifier) @name
- )
-) @reference.send
-
-(namespace_declaration
- name: (identifier) @name
-) @definition.module
+(class_declaration name: (identifier) @name) @definition.class
+
+(class_declaration bases: (base_list (_) @name)) @reference.class
+
+(interface_declaration name: (identifier) @name) @definition.interface
+
+(interface_declaration bases: (base_list (_) @name)) @reference.interface
+
+(method_declaration name: (identifier) @name) @definition.method
+
+(object_creation_expression type: (identifier) @name) @reference.class
+
+(type_parameter_constraints_clause target: (identifier) @name) @reference.class
+
+(type_constraint type: (identifier) @name) @reference.class
+
+(variable_declaration type: (identifier) @name) @reference.class
+
+(invocation_expression function: (member_access_expression name: (identifier) @name)) @reference.send
+
+(namespace_declaration name: (identifier) @name) @definition.module
+
+(namespace_declaration name: (identifier) @name) @module
```

Comment:
Is this the only new line? Looks same as definition above with `@definition` removed.

What does this do?

Reply:
This adds a new tag to namespace declarations, so that now they'll be tagged as both `definition.module` and `module`. We're only adding rather than replacing because we don't want to cause any problems for anyone who might be relying on the `definition.module` tag.